tests/test_regrid.py

```diff
@@ -97,46 +72,60 @@ def conservative_sample_grid():
     return xarray_regrid.create_regridding_dataset(grid)
 
 
-def test_conservative_regridder(conservative_input_data, conservative_sample_grid):
+@pytest.mark.parametrize("method", ["linear", "nearest"])
+def test_basic_regridders_ds(
+    sample_input_data, sample_grid_ds, cdo_comparison_data, method
+):
+    """Test the dataset regridders (except conservative)."""
+    regridder = getattr(sample_input_data.regrid, method)
+    ds_regrid = regridder(sample_grid_ds)
+    ds_cdo = cdo_comparison_data[method]
+    xr.testing.assert_allclose(ds_regrid, ds_cdo, rtol=0.002, atol=2e-5)
```

Comment:
After padding I was able to remove the edge selection in these and still get the tests to pass with a slight reduction in the tolerance.

Reply:
Awesome! :smile: